charts/hobbyfarm/crds/virtualmachineset.yaml

```diff
@@ -8,4 +8,9 @@ spec:
   names:
     kind: VirtualMachineSet
     plural: virtualmachinesets
+    singular: virtualmachineset
+    shortNames:
+    - vms
```

Comment:
Should this be a name for `virtualmachines`? Might be confusing to put it here.

Reply:
Fair, it can be confusing, although I have been using that shorthand extensively up to now.